src/test/scala/test/SeoSanityTest.scala

```diff
@@ -0,0 +1,54 @@
+package test
+
+import org.scalatest.FlatSpec
+import org.scalatest.matchers.ShouldMatchers
+import play.api.libs.json._
+
+class SeoSanityTest extends FlatSpec with ShouldMatchers with Http {
+
+  def checkUrl(url: String): Unit = {
+    val connection = GET(
+      s"http://linter.structured-data.org/?url=$url",
+      compress = true,
+      headers = Seq(
+        "Accept" -> "application/json"
+      )
+    )
+
+    val json: JsValue = Json.parse(connection.body)
+
+    val messages = (json \ "messages").as[JsArray].value
```

Comment:
You could get the value as an `Opt[JsArray]` then use [OptionValues](http://doc.scalatest.org/2.2.1/#org.scalatest.OptionValues) to assert on it. That way if the JSON comes back in an unexpected structure, the test will fail with a clearer status and throw `TestFailedException` rather than throwing an unexpected exception.